### PR TITLE
added program title to terminal title

### PR DESCRIPTION
**Pull request by Brobin**
Displays the name and version in the terminal title. Nothing too fancy, just helps when you have 5 terminals open :D

Tested on linux. The first method works for linux and OSX. The second should work for Windows..

### CODE REVIEW

File: rtv/__main__.py [hunk 1]
```diff
@@ -68,6 +69,13 @@ def main():
     args = command_line()
     local_config = load_config()
 
+    # set the terminal title
+    title = 'rtv {0}'.format(__version__)
+    try:
+        sys.stdout.write("\x1b]2;{0}\x07".format(title))
```
Comment (michael-lazar) [inline rtv/__main__.py]: Do you have a source that explains how this works?

Comment (Brobin) [inline rtv/__main__.py]: I found it on the [arch wiki](https://bbs.archlinux.org/viewtopic.php?id=85567).

Windows method (and linux again) on this [stackoverflow post](http://stackoverflow.com/questions/2330393/how-to-set-the-program-title-in-python#answer-2330596)

Comment (michael-lazar) [inline rtv/__main__.py]: Interesting. I think based on that link the windows method should be 
```python
os.system('title {}'.format(title))
```

Comment (Brobin) [inline rtv/__main__.py]: My mistake I'll fix that real quick.

**Comment by Brobin** [comment]
I also just make it check `os.name` for the windows version because I don't think `sys.stdout` would throw an exception in Windows.

**Pull request merged**